_data/people.yml

```diff
@@ -1,30 +1,63 @@
-# A 'database' of people participating in the JLESC cooperation
 #
-# Each entry consists of a unique identifier and some fields:
+# Database of People participating in JLESC
 #
-#     identifier:
-#       sur_name: Doe
-#       given_name: John
-#       affiliation: affiliation_id
-#       position: Head of Department
-#       topics: awesome research
-#       email: official email
-#       homepage: link to homepage
 #
-# The identifier should be constructed from the surname and the initial letter of the first name,
-# all lower case and separated by an underscore.
+# ## Purpose
 #
-# The affiliation must be an affiliation identifier as found in /_data/institutes.yml.
-# Multiple affiliations can be given in list/array syntax:
-#    [first, second]
+# - data source for the table on `about/people.html`
+# - data source for custom _person_ Liquid tags
 #
-# The email should be the official 'work' email of the person, e.g. @fz-juelich.de.
 #
-# The homepage field can be left empty, but should point to a personal page with more details on
-# research interests, publications and contact data.
+# ## Identifier Format
 #
-# !! NOTE !!
-# Please keep the entries below in alphanumerical order according to the identifier. Thanks!
+# For the sake of uniqueness and uniformity, construct a person's identifier according to this
+# scheme:
+#
+#   SURNAME1_SURNAME2_GIVEN_NAME_INITIAL
+#
```

Comment:
Just to be clear: `GIVEN_NAME_INITIAL` means "first character of the first given name".